File: CS/WebSite/App_Code/Article.cs
using System;
using DevExpress.Xpo;

public class Article: XPCustomObject {
    public Article(Session session)
        : base(session) { }

    private Guid _UniqueID;
    [Key(true)]
    public Guid UniqueID {
        get { return _UniqueID; }
        set { SetPropertyValue("UniqueID", ref _UniqueID, value); }
    }

    private Category _Category;
    [Association("Category-Articles")]
    public Category Category {
        get { return _Category; }
        set { SetPropertyValue("Category", ref _Category, value); }
    }

    private string _Name;
    public string Name {
        get { return _Name; }
        set { SetPropertyValue("Name", ref _Name, value); }
    }
}

File: VB/WebSite/App_Code/Article.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.Xpo

Public Class Article
	Inherits XPCustomObject
	Public Sub New(ByVal session As Session)
		MyBase.New(session)
	End Sub

	Private _UniqueID As Guid
	<Key(True)> _
	Public Property UniqueID() As Guid
		Get
			Return _UniqueID
		End Get
		Set(ByVal value As Guid)
			SetPropertyValue("UniqueID", _UniqueID, value)
		End Set
	End Property

	Private _Category As Category
	<Association("Category-Articles")> _
	Public Property Category() As Category
		Get
			Return _Category
		End Get
		Set(ByVal value As Category)
			SetPropertyValue("Category", _Category, value)
		End Set
	End Property

	Private _Name As String
	Public Property Name() As String
		Get
			Return _Name
		End Get
		Set(ByVal value As String)
			SetPropertyValue("Name", _Name, value)
		End Set
	End Property
End Class